assets/models/system/deci-decidiffusion-v1-0/spec.yaml

```diff
@@ -0,0 +1,36 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: deci-decidiffusion-v1-0
+path: ./
+properties:
+  SHA: 10c31ce02b006ae5ceb774dcafdc3a0bcba992ef
+  datasets: LAION-v2
+  inference-min-sku-spec: 4|1|28|176
+  inference-recommended-sku: Standard_NC6s_v3, Standard_NC12s_v3, Standard_NC24s_v3,
+    Standard_NC24rs_v3, Standard_NC16as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4,
+    Standard_NC4as_T4_v3, Standard_NC64as_T4_v3, Standard_NC8as_T4_v3, Standard_NC96ads_A100_v4,
+    Standard_ND40rs_v2, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4
+tags:
+  Preview: ''
+  license: creativeml-openrail++-m
+  task: text-to-image
+  author: deci-ai
+  huggingface_model_id: Deci/DeciDiffusion-v1-0
+  inference_compute_allow_list:
+    [
```

Comment:
Do we need the square brackets in the yaml?

Reply:
yes, accroding to this https://github.com/Azure/azureml-assets/blob/5b64288021fcbcec8e4e74db2282e7a959c84b1f/assets/models/system/microsoft-orca-2-13b/spec.yaml